.github/workflows/publish-dts-sdk.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish Durable Task Scheduler to PyPI
+
+on:
+  push:
+    tags:
+      - 'v*'  # Only run for tags starting with "v"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Extract version from tag
+        run: echo "VERSION=${GITHUB_REF#refs/tags/v}" >> $GITHUB_ENV  # Extract version from the tag
```

Comment:
```suggestion
        run: echo "VERSION=${GITHUB_REF#refs/tags/azuremanaged-v}" >> $GITHUB_ENV  # Extract version from the tag
```

Reply:
FYI - right now we are not doing anything with the version. However, if we wanted to - we could use this version from the tag and DYNAMICALLY replace the value in pyproject.toml BEFORE running the build and publish commands. This way releasing a new SDK only requires tagging... otherwise what would be required is also a manual PR to change the version in pyproject -- and in fact what can happen is that the pyproject file version and the tag version are not aligned.

Consider having a script here that uses `sed` or similar to replace the version within the `pyproject.toml`